OptionSelector/OptionsTableViewController.m

```diff
@@ -0,0 +1,126 @@
+//
+//  OptionsTableViewController.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OptionsTableViewController.h"
+#import "CategoryModel.h"
+#import "CQCategory.h"
+
+@interface OptionsTableViewController ()
+
+@end
+
+@implementation OptionsTableViewController
+
+- (void)viewDidLoad {
```

Comment:
Style: Any time you have an empty method definition or one that only calls `super`, you can remove the method definition entirely.